LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2021 EasyPost
+Copyright (c) 2022 EasyPost
```

Comment:
TBF, we made this library before 2021, so clearly we've updated the copyright year before.

Reply:
Indeed. Per an internal convo, let's reset this Copyright year to whenever we first released the library (as copyright is inherently a reality from that date forward. By updating the year it could be construed that we nullify the copyright prior to that date, additionally, I don't want to have to update the date each year across 7 languages.

Reply:
Sounds good, resolved in a56777c